apps/channels/models.py

```diff
@@ -5,14 +5,16 @@
 from django.db import models
 from django.db.models import JSONField
 from django.urls import reverse
-from telebot import TeleBot, apihelper
+from telebot import TeleBot, apihelper, types
 
 from apps.experiments.models import Experiment, ExperimentSession
 from apps.utils.models import BaseModel
 from apps.web.meta import absolute_url
 
 
 class ExperimentChannel(BaseModel):
+    RESET_COMMAND = "/reset"
```

Comment:
Going back on my previous comment about commands: I did a bit of digging and `/reset` is not a telegram command. Bot's by default don't have commands, so we can 1. use this command and 2. register it as a bot command, so when the user types "/" (or click on the new Menu button), the list of available commands with a description will be shown. Example:
![image](https://github.com/dimagi/open-chat-studio/assets/64970009/e0c9d99c-c593-435a-9b85-db360ffcb89f)

Reply:
cc @bderenzi

Reply:
Cool, I'm happy with `/reset`. And +1 on being able to expand it in the future to have additional aliases. I'm not sure `/reset` will be the most obvious for young people in Kenya and certainly wouldn't expect that to be true in Senegal.

Reply:
We could maintain a list of reset commands in different languages (maintenance burden), or ask the bot to tell us if the user wants to reset the chat (token burden).

Reply:
I think I'm pretty against asking the bot to tell us. I'd say let's defer any further action until it becomes a problem.

Reply:
agreed with both.